tests/test_item.rs

```diff
@@ -514,3 +514,43 @@ fn test_type() {
         })
     );
 }
+
+#[test]
+fn test_impl() {
+    let builder = AstBuilder::new();
+    let impl_ = builder.item().impl_()
+        .trait_().id("ser::Serialize")
```

Comment:
this should be `.path().id("ser").id("Serialize").build()` or `.path().ids(&["ser", "Serialize"]).build()`